void function returning a value

doxygen gives the warnings:
```
.../ITK/Modules/Core/FiniteDifference/include/itkFiniteDifferenceImageFilter.h:180: warning: found documented return type for itk::FiniteDifferenceImageFilter::SetDifferenceFunction that does not return anything
.../ITK/Modules/Core/GPUFiniteDifference/include/itkGPUFiniteDifferenceImageFilter.h:89: warning: found documented return type for itk::GPUFiniteDifferenceImageFilter::SetDifferenceFunction that does not return anything
```

The functions are probably `void` functions and cannot have a return value (Maybe a value is returned as an argument)

--- a/Modules/Core/FiniteDifference/include/itkFiniteDifferenceImageFilter.h
+++ b/Modules/Core/FiniteDifference/include/itkFiniteDifferenceImageFilter.h
@@ -178,8 +178,7 @@ public:
   itkGetConstReferenceObjectMacro(DifferenceFunction, FiniteDifferenceFunctionType);
 
   /** This method sets the pointer to a FiniteDifferenceFunction object that
-   * will be used by the filter to calculate updates at image pixels.
-   * \returns A FiniteDifferenceObject pointer. */
+   * will be used by the filter to calculate updates at image pixels. */
   itkSetObjectMacro(DifferenceFunction, FiniteDifferenceFunctionType);
 
   /** Set/Get the number of iterations that the filter will run. */

--- a/Modules/Core/GPUFiniteDifference/include/itkGPUFiniteDifferenceImageFilter.h
+++ b/Modules/Core/GPUFiniteDifference/include/itkGPUFiniteDifferenceImageFilter.h
@@ -87,8 +87,7 @@ public:
   }
 
   /** This method sets the pointer to a FiniteDifferenceFunction object that
-   * will be used by the filter to calculate updates at image pixels.
-   * \returns A FiniteDifferenceObject pointer. */
+   * will be used by the filter to calculate updates at image pixels. */
   void
   SetDifferenceFunction(FiniteDifferenceFunctionType * differenceFunction) override
   {

